src/libstore/remote-store.hh

```diff
@@ -171,6 +176,9 @@ private:
 
     std::atomic_bool failed{false};
 
+    void copyDrvsFromEvalStore(
+        const std::vector<DerivedPath> & paths,
+        std::shared_ptr<Store> evalStore);
```

Comment:
I've been meaning to bring this up, can we call it "DrvStore" or similar in libstore? "Eval" is not a concept libstore should be aware of.

Also, can we do
```suggestion
    void copyDrvsFromEvalStore(
        const std::vector<DerivedPath> & paths,
        Store & evalStore);
```
I am hoping to cut down on the extraneous use of shared pointers (for ergonomics, more than efficiency).

Reply:
The only problem with that is that if you need a `shared_ptr`, it's hard to get it from a `&`.